packages/cli/lib/projects.js

```diff
@@ -807,6 +810,14 @@ const createProjectComponent = async (component, name) => {
   );
 };
 
+const showPlatformVersionWarning = platformVersion => {
```

Comment:
This util would probably be a little more useful if you set it up to just take in `projectConfig`. Then none of the consuming commands will be required to know the name of the `platformVersion` field. That would make it easier to change the name of the platform version field down the road if we ever decide to do that. There will be less placed in the code that directly reference it.

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/885/commits/36ba46aa3a3f52236a7629e31d9c7167b626a156